scene/debugger/scene_debugger.cpp

```diff
@@ -127,6 +133,43 @@ Error SceneDebugger::parse_message(void *p_user, const String &p_msg, const Arra
 			scene_tree->get_root()->set_camera_3d_override_orthogonal(size_or_fov, depth_near, depth_far);
 		}
 		scene_tree->get_root()->set_camera_3d_override_transform(transform);
+
+	} else if (p_msg == "add_preview_nodes_to_current_3d_scene") {
+		ERR_FAIL_COND_V(p_args.size() < 6, ERR_INVALID_DATA);
+		const bool should_add_world_environment = p_args[0];
+		const bool should_add_directional_light_3d = p_args[1];
+		const bool should_add_camera_3d = p_args[2];
+
+		if (should_add_world_environment) {
+			WARN_PRINT_ED("Current scene was run without a WorldEnvironment node, adding preview WorldEnvironment to the scene. You can disable this under Project Settings.");
```

Comment:
For this and the other messages, please add the path of the setting (`editor/run/automatically_add_preview_nodes_when_running_single_3d_scene`) so users can simply search for this string to find the setting in the UI.

Reply:
Done.

Reply:
Was wondering why that was failing, thanks man.